front/packages/ui/src/login/login.tsx

```diff
@@ -53,43 +57,47 @@ export const LoginPage: QueryPage<{ apiUrl?: string; error?: string }> = ({
 	return (
 		<FormPage apiUrl={apiUrl}>
 			<H1>{t("login.login")}</H1>
-			<OidcLogin apiUrl={apiUrl} />
-			<P {...css({ paddingLeft: ts(1) })}>{t("login.username")}</P>
-			<Input
-				autoComplete="username"
-				variant="big"
-				onChangeText={(value) => setUsername(value)}
-				autoCapitalize="none"
-			/>
-			<P {...css({ paddingLeft: ts(1) })}>{t("login.password")}</P>
-			<PasswordInput
-				autoComplete="password"
-				variant="big"
-				onChangeText={(value) => setPassword(value)}
-			/>
-			{error && <P {...css({ color: (theme) => theme.colors.red })}>{error}</P>}
-			<Button
-				text={t("login.login")}
-				onPress={async () => {
-					const { error } = await login("login", {
-						username,
-						password,
-						apiUrl,
-					});
-					setError(error);
-					if (error) return;
-					router.replace("/", undefined, {
-						experimental: { nativeBehavior: "stack-replace", isNestedNavigator: false },
-					});
-				}}
-				{...css({
-					m: ts(1),
-					width: px(250),
-					maxWidth: percent(100),
-					alignSelf: "center",
-					mY: ts(3),
-				})}
-			/>
+			<OidcLogin apiUrl={apiUrl} hideOr={!data?.passwordLoginEnabled} />
+			{data?.passwordLoginEnabled && (
+				<Array>
```

Comment:
woaw didn't even know `<Array>` was a thing (ig it works because it's just calling js's `new Array()` method).

Could you use fragments instead? (that's just `<>` & `</>`)

Reply:
sure - done